packngo.go

```diff
@@ -613,6 +622,12 @@ func checkResponse(r *http.Response) error {
 		return err
 	}
 
+	ct := responseContentType(r)
+	if ct != expectedAPIContentType {
```

Comment:
I don't know if we can be sure this will stay precisely the same.

I think we can safely expect `application/json`.   (https://www.ietf.org/rfc/rfc4627.txt)
But there could be options in addition to `charset=utf-8` and those options could come in any order.
I wouldn't want packngo to break because new options were added.

Reply:
See your captured response in https://github.com/packethost/terraform-provider-packet/issues/269 for an example where `charset` was not included.

Reply:
I looked for a helper in `http`, but I couldn't find any that would parse the content-type.

Reply:
nice catch. What about `strings.HasPrefix(ct, "application/json")` or  `strings.Contains(ct, "application/json")` ?

Reply:
I was thinking `HasPrefix`. I don't think the spec allows for "application/json" to appear anywhere but the beginning of the string.

Reply:
I used HasPrefix